.gitignore

```diff
@@ -1,2 +1,6 @@
 .gitignore
 .idea/
+src/muqsit
+src/poggit
+src/spoondetector
+  
```

Comment:
This change to `.gitignore` shouldn't be required since poggit virions don't directly add the library into the plugin, they rather shard it's namespace. For those who run it through source code, virions would be present in `/path/to/server/directory/virions/`.